README.md

```diff
@@ -1,655 +1,288 @@
+* **Text justification.** Use `pos` to declare whether you want the text to appear left justified (`pos = 4`), right justified (`pos=2`), or centered (`pos=1` or `pos=3`).
+* **Text location.** Adjust `x_loc` and `y_loc` to change where the text appears. The value of `pos` will matter here since pos changes whether your text is below (`pos=1`), left of (`pos=2`), above (`pos=3`), or right of (`pos=4`) the x and y location you specify. The x and y location values are the fraction corresponding to the full image, starting from the bottomleft. See `6_visualize/in/frame_grid_xy_callout_locations.png` for some helpful positions of x and y location values and where they appear on the animation frame.
+* **Event type.** Each event callout is given a box indicating the event duration on the date wheel visual on the bottomleft of the frame. By default, events are given a blue color (`"#04507d"`) for `wheel_color`. Update the event to a red color (`"#ca0020"`) if the event describes a drought or dry event.
+
+These are the only elements you need to change right now. We will iterate on the others later.
+
+*You may be wondering what the `polygon` element under each callout is for ... don't worry about this and leave as the default. We will hopefully add information about those at a later date.*
 
-run_magick_cmd("convert -size 11400x3721 canvas:white carousel_background.png")
-run_magick_cmd(sprintf("convert -composite -gravity center carousel_background.png %s %s_carousel.png", frame_to_use, version_info))
+#### Rebuild the frames and animation with the callouts
 
+Using the initial callouts timing and placement, generate a new animation by updating the appropriate frames and then rebuilding the video using the code below.
+
+```r
+source('helper_fxns_pipeline.R')
+rebuild_frame_sections(timestep = TRUE, pause = TRUE)
+rebuild_video()
 ```
 
-# To create a Drupal thumbnail-optimized image, run the following
+Sometimes, I notice that the `pause` frames are not updating as they should. There are not many of them, so if this happens I delete the individual frames manually and then rebuild. To delete manually, find any frame in `6_visualize/tmp` prefixed with `frame_6000`.
+
+### Tweak event callout timing and appearance
+
+This is usually where I spend the most hands-on time because it is where you iterate on how the callouts integrate with the animation. Usually, the first animation is not ready for publication because callouts may overlap each other visually or temporally, some may extend the full length of the animation which isn't very useful, and others may blip on and then off too quickly for a user to read. Below we detail the other attributes and methods that you can use to polish the final visualization.
```

Comment:
These elements will have defaults based on the callout_cfg.yml. I think this is the appropriate place for this information because you should build the animation once all the way through using the callout defaults before deciding how to tweak them.